backend/helpers/middlewares/auth.js

```diff
@@ -37,5 +37,18 @@ const authMiddleware = async (req, res, next) => {
   res.locals.decode = payload;
   next();
 };
-
-module.exports = { authMiddleware, generateJWT, verifyToken };
+function FetchUser(req, res, next) {
```

Comment:
[question] Why did we need to introduce this function, why can't be use the `authMiddleware` defined above in this file?

Reply:
I was not using bearer token. So i was getting an error. So i had created this function but now i have removed this fetchuser function and using authMiddleware.

Reply:
yes

Reply:
![Uploading Screenshot (714).png…]()